app/frontend/src/features/communities/discussions/DiscussionsSection.tsx

```diff
@@ -54,6 +56,17 @@ export default function DiscussionsSection({
       {discussionsError && (
         <Alert severity="error">{discussionsError.message}</Alert>
       )}
+      <Link
+        to={`${routeToCommunity(
+          community.communityId,
+          community.slug,
+          "discussions"
+        )}#${composingDiscussionHash}`}
+      >
+        <Button size="small" className={classes.newPostButton}>
+          {NEW_POST_LABEL}
+        </Button>
+      </Link>
```

Comment:
I think this will lead to the warning about invalid nesting of "interactive elements" under an anchor and so leads to invalid HTML. Probably should make this a link only since it goes to a new URL?

Reply:
Link looks super weird. I made it a span